.github/workflows/mysql-migtests.yml

```diff
@@ -30,7 +33,7 @@ jobs:
     - name: Fetch debezium-server
       uses: docker://amazon/aws-cli:2.7.27
       with:
-        args: s3 cp s3://voyager-automation-data/debezium-latest.tar.gz .
+        args: s3 cp s3://voyager-automation-data/debezium-server-dist-93c7704d1fe662d515c48cf58feadf49168b33f0.tar.gz .
```

Comment:
We should not need to update this file each time we update the debezium tar file. I would suggest to keep the `latest` as a placeholder to mean the most recently uploaded tar file.

Reply:
cc @makalaaneesh

Reply:
Makes sense. Tomorrow when we automate debezium build -> S3, the automation tests will run against latest commit on debezium/voyager-main. 

I was just thinking it would be useful to know which commit the tests have run against. Probably that can be solved in a different manner.